.github/workflows/ci.yml

```diff
@@ -96,6 +96,16 @@ jobs:
         env:
           ANSYSLMD_LICENSE_FILE: 1055@${{ secrets.LICENSE_SERVER }}
 
+      - name: Upload coverage artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.python-version }}_${{ matrix.os }}_pytest.xml
+          path: .cov/coverage.xml 
```

Comment:
Shouldn't it be html here?

Reply:
Yes, Here the pytest is running with pyansys/actions with extra args , not from toml, adapting to html everywhere.